### DISCUSSION TITLE
cistarget databases for run_pycistarget()

### DISCUSSION THREAD

**Discussion by jesswhitts** [aertslab/scenicplus#231] [Q&A] [answered]
Hi, 

I am working through the MM cell line tutorial, and have struggled to create the cistarget database. Can you provide any more specific details on how this was done? Or some example code please?

Also, I have downloaded a 'generic' cistarget database from here: 
https://resources.aertslab.org/cistarget/databases/homo_sapiens/hg38/refseq_r80/mc_v10_clust/gene_based/

But I get an error, it seems incompatible somehow. Am I using the right files?

This is the error, when reading ctx_db:
ValueError: Length mismatch: Expected axis has 0 elements, new values have 3 elements

Thanks,
jess

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @jesswhitts 

For creating the cistarget database please see the Readme.md file on https://github.com/aertslab/create_cisTarget_databases

The "generic" database you downloaded is the wrong one, you want a region-based database while you downloaded a gene-based one. The correct one can be found here: https://resources.aertslab.org/cistarget/databases/homo_sapiens/hg38/screen/mc_v10_clust/region_based/

I hope this helps.

Best,

Seppe

**Reply by jesswhitts** [reply]
This is very helpful, thank you @SeppeDeWinter !
